packages/runtime-core/src/components/KeepAlive.ts

```diff
@@ -6,7 +6,8 @@ import {
   LifecycleHooks,
   currentInstance,
   getComponentName,
-  ComponentOptions
+  ComponentOptions,
+  formatComponentName
```

Comment:
`formatComponentName` is a dev only function and should not be used in prod code paths.

The proper fix probably requires automatically setting the `name` option from inferred filename.